File: Java/0002.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        String[] numbers = input.split(" ");

        int a = Integer.parseInt(numbers[0]);
        int b = Integer.parseInt(numbers[1]);

        String x;
        if (a > b) {
            x = ">";
        } else if (a == b) {
            x = "=";
        } else {
            x = "<";
        }

        System.out.println(x);
        scanner.close();
    }
}

File: Kotlin/0002.kt
import java.util.Scanner

fun main() {
    val scanner = Scanner(System.`in`)
    val input = scanner.nextLine()
    val numbers = input.split(" ")

    val a = numbers[0].toInt()
    val b = numbers[1].toInt()

    val x = if (a > b) {
        ">"
    } else if (a == b) {
        "="
    } else {
        "<"
    }

    println(x)
}